### DISCUSSION TITLE
Keep uptodate with master without Ultralisp

### DISCUSSION THREAD

**Discussion by mmontone** [neomacs-project/neomacs#58] [Ideas]
I keep up to date with neomacs source without using Ultralisp. What I do is point ASDF to dependencies shipped in the [releases](https://github.com/neomacs-project/neomacs/releases).

Here is my script, the directories need to be adapted, though:

```lisp
(load "/home/marian/src/lisp/asdf.lisp")

(asdf:initialize-source-registry
  '(:source-registry
    :ignore-inherited-configuration
    (:tree "/home/marian/src/neomacs/") ;; directory with neomacs source code (git master)
    (:tree "/home/marian/src/neomacs-bin/bin/src/") ;; directory with neomacs binary release
    (:tree "/home/marian/quicklisp/dists/quicklisp/software/")))

(asdf:load-system :neomacs)

(neomacs:start nil)
```

Then I invoke: `sbcl --no-userinit --load load.lisp`

I share in case someone else is in same situation, that find it difficult to compile via Ultralisp.

**Comment by vindarel** [comment]
Thanks. BTW, I found it OK to load without Ultralisp, I only had to clone lwcells into the local-projects. (https://github.com/kchanqvq/lwcells)

**Reply by mmontone** [reply]
Ah. Good to know! I've tried to setup a qlot build but my QLot is failing with a possible bug. Would you like to try yourself?

**Reply by vindarel** [reply]
in a couple days… maybe!
